continuous_integration/recipe/meta.yaml

```
@@ -22,7 +22,7 @@ build:
 
 requirements:
   build:
-    - rust>=1.62.1
+    - {{ compiler('rust') }}
     - setuptools-rust>=1.4.1
```

Would space this out and the one below. Know it isn't affected by this change. Just noticed it and though this would be a good opportunity to fix it

```suggestion
    - setuptools-rust >=1.4.1
```